admin/includes/init_includes/init_templates.php

```diff
@@ -43,5 +43,9 @@
 if ($pagename == '') {
   $pagename = STORE_NAME;
 }
-$title = TEXT_ADMIN_TAB_PREFIX . ' ' . $pagename;
+if (strncmp(TEXT_ADMIN_TAB_PREFIX, $pagename, strlen(TEXT_ADMIN_TAB_PREFIX)) == 0) {
+   $title = $pagename;
+} else {
+   $title = TEXT_ADMIN_TAB_PREFIX . ' ' . $pagename;
+}
```

Comment:
Isn't the current recommendation for if/else style code to keep the original line but if it meets the proposed condition reassign it?
I.e.
```diff
$title = TEXT_ADMIN_TAB_PREFIX . ' ' . $pagename;
+if (strncmp(TEXT_ADMIN_TAB_PREFIX, $pagename, strlen(TEXT_ADMIN_TAB_PREFIX)) == 0) {
+    $title = $pagename;
+}
```

Reply:
If you got to the home page and look at the title in your browser tab you'll see it says "Admin Admin Home."  That was the driver for this change.

Reply:
Ok, I do think I've seen it, but thought it "had always been that way". :)